starknet_programs/cairo1/multi_syscall_test.cairo

```diff
@@ -0,0 +1,108 @@
+#[contract]
+mod multy_syscall {
+    use starknet::get_caller_address;
+    use starknet::get_contract_address;
+    use starknet::get_execution_info_syscall;
+    use starknet::replace_class_syscall;
+    use starknet::library_call_syscall;
+    use starknet::call_contract_syscall;
+    use starknet::send_message_to_l1_syscall;
+    use starknet::storage_read_syscall;
+    use starknet::storage_write_syscall;
+    use starknet::deploy_syscall;
+
+    use starknet::storage_access::{StorageAddress, storage_address_try_from_felt252};
+    use starknet::contract_address::ContractAddress;
+    use array::{Array, ArrayTrait, Span, SpanTrait};
+    use result::ResultTrait;
+    use option::OptionTrait;
+    use starknet::class_hash::ClassHash;
+    use traits::TryInto;
+    use starknet::class_hash::Felt252TryIntoClassHash;
+    use box::Box;
+    use traits::Into;
+    use box::BoxTrait;
+    use zeroable::Zeroable;
+   
+    #[external]
+    fn caller_address() -> ContractAddress {
+        get_caller_address()
+    }
+
+     #[external]
+    fn contract_address() ->  ContractAddress {
+        get_contract_address()
+    }
+
+     #[external]
+    fn execution_info_syscall() {
+        get_execution_info_syscall();
+    } 
```

Comment:
Please try to return some fields of the execution info so you can test it is returning the right values